integration-tests/src/test/java/org/apache/druid/tests/coordinator/duty/ITAutoCompactionTest.java

```diff
@@ -1829,8 +1835,9 @@ private void submitCompactionConfig(
         transformSpec,
         !dropExisting ? null : new UserCompactionTaskIOConfig(true),
         engine,
-        null
+        ImmutableMap.of("maxNumTasks", 2)
     );
+    LOG.info("Submitting compaction config for engine[%s].", engine == null ? CompactionEngine.NATIVE: engine);
```

Comment:
Is this still needed?

Reply:
Added this to be able to make out which engine was used to run the test, since this info is present nowhere else in the tests logs -- unless you look at the task logs.

Reply:
Hmm, I think it is already logged as a part of the current test name itself. Something like:

```
2024-07-22T17:14:55,292 INFO [main] org.apache.druid.testing.utils.LoggerListener - Starting [org.apache.druid.tests.coordinator.duty.ITAutoCompactionTest.testAutoCompactionDutyCanDeleteCompactionConfig] with parameters [native]
```
https://github.com/apache/druid/actions/runs/10044009147/job/27761396030?pr=16768#step:6:23873

Reply:
Thanks for pointing it out. I don't see the above line while running them on local for some reason. Have anyway reverted the log I added.